src/Kunstmaan/NodeBundle/Entity/AbstractPage.php

```diff
@@ -128,11 +128,10 @@ public function getDefaultAdminType()
     /**
      * {@inheritdoc}
      *
-     * @deprecated Using the service action is deprecated in KunstmaanNodeBundle 5.1 and will be removed in KunstmaanNodeBundle 6.0. Use the SlugActionInterface instead
+     * @deprecated Using the service method is deprecated in KunstmaanNodeBundle 5.1 and will be removed in KunstmaanNodeBundle 6.0. Implement SlugActionInterface and use the getControllerAction method to provide custom logic instead.
      */
     public function service(ContainerInterface $container, Request $request, RenderContext $context)
     {
-        @trigger_error('Using the service action is deprecated in KunstmaanNodeBundle 5.1 and will be removed in KunstmaanNodeBundle 6.0. Use the SlugActionInterface instead', E_USER_DEPRECATED);
```

Comment:
Remove the deprecation here because it would trigger a deprecation warning even if the user has not defined the method in their child class.